doc/guide/packages.xml

```diff
@@ -186,6 +186,39 @@ $ cockpit-bridge --packages
         <listitem><para>The relative path to the HTML file within the package that implements
           the menu item or tool.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>keywords</term>
+        <listitem><para>Keywords that describe the page and which are used for searching.
+          These keywords should be lowercase. Page label is prepended as first keyword.
+          Keywords is object of keyword items - content of such item is described below.
+        </para></listitem>
+      </varlistentry>
+    </variablelist>
+
+    <para>Keyword items are registered using JSON objects that have the
+      following properties:</para>
+
+    <variablelist>
+      <varlistentry>
+        <term>matches</term>
+        <listitem><para>List of keywords to be matched.</para></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>goto</term>
+        <listitem><para>Optional path that is used for all keywords in this item.
+          When this argument starts with slash, then it is used as pathname, otherwise
+          it is used as hash.</para></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>coef</term>
+        <listitem><para>How much keywords are prioritized over other. Default is 3.
```

Comment:
> I suppose it's for conflict resolution when more than one page has the same keyword

It is for two things:
1. Finding the best keyword on a page (so you can have some keywords which are more important and will be preferred [example is that manifest is preferred to what page provides])
2. Ordering pages based on the best result (so each page finds the best keyword and provides its score - and then are ordered based on this score)

This `coef` is not score itself, it is just value that is added to the score calculated (if the keyword starts with the term then it has bigger score as if the term is just inside and also when the keyword native version matches it has bigger priority to English version matching). But I can call it priority.

> Also, we don't currently use this, so perhaps it shouldn't be documented yet until we actually need the feature?

Well I think since it can be used and code would consider it I think it should be documented. Also we won't forget to document it in the future. I just did not use it in any manifest, since I was happy with the default value.

Reply:
Right, it won't be the final score, I saw the function that adjusts it. It would still not be a bad name for it IMHO as it's the part of the scoring that the manifest author decides upon.

"weight" may be another alternative. </bikeshed>